modules/repository-datastax/src/test/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepositoryTest.java

```diff
@@ -722,4 +723,31 @@ public void testGetTagList() throws ExecutionException, InterruptedException {
         assertArrayEquals(exp2.toArray(), callResult.get(app02).toArray());
     }
 
+
+    @Test
+    public void testGetEmptyTagList() throws ExecutionException, InterruptedException {
+        //------ Input --------
+        Application.Name app01 = Application.Name.valueOf("app01");
+        List<Application.Name> appNames = Arrays.asList(app01);
+
+        List<ExperimentTagsByApplication> exp1Tags = Collections.emptyList();
+
+        //------ Mocking interacting calls
+        ListenableFuture<Result<ExperimentTagsByApplication>> dbResultFuture1 = mock(ListenableFuture.class);
+
+        Result<ExperimentTagsByApplication> dbResult1 = mock(Result.class);
```

Comment:
@AndreaSuckro Let us just keep it as `dbResult` or rather `dbResultMock` instead of `dbResult1`.